### PR TITLE
Adding HCal FB LUT to caloParams for uploading to DB

### PR DESCRIPTION
**Pull request by vshang**
#### PR description:

This PR is to add a parameter to the latest caloParams file (caloParams_2022_v0_6_cfi.py) containing the HCal FB LUT for the LLP trigger as well as modifications to CaloL1 software to handle converting the HCal FB LUT from the caloParams file to the CaloL1 Algo settings key to upload to the DB.

#### PR validation:

Changes were tested locally on CMSSW_13_1_0_pre2 release by using the modified caloParams file to produce a CaloL1 Algo settings key containing the correct HCal FB LUT. This Algo settings key has been tested and validated during several P5 tests together with the new CaloLayer1 firmware and SWATCH software to upload the HCal FB LUT from DB to firmware.

### CODE REVIEW

**Comment by aloeliger** [comment]
@vshang are all these commits intended to be here? I see one from @pallabidas for the L1TCaloSummary plugin that shouldn't be touched for HCal FB LUT changing. I see several from @bundocka and @epalencia as well. Was `git cms-merge-topic` used to create this PR?

**Comment by vshang** [comment]
@aloeliger Ah, seems like I forgot to remove the commits after merging l1t-integration-v147-CMSSW_13_1_0_pre2 branch while testing the code. Do you want me to make a new PR from CMSSW_13_1_0_pre2 with just the commits I made for the HCAL FB LUT changes?

**Comment by aloeliger** [comment]
> @aloeliger Ah, seems like I forgot to remove the commits after merging l1t-integration-v147-CMSSW_13_1_0_pre2 branch while testing the code. Do you want me to make a new PR from CMSSW_13_1_0_pre2 with just the commits I made for the HCAL FB LUT changes?

@vshang Yes please.

File: EventFilter/GEMRawToDigi/test/runGEMUnpacker_cfg.py [hunk 1]
```diff
@@ -159,6 +159,11 @@
     )
     process.muonGEMDigis.useDBEMap = True
 
+## for the time being the mapping does not work with the data label. Use MC instead
+if options.useB904Data:
+    process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:phase1_2022_realistic', '')
+>>>>>>> a607580e7d2 (Rename phase1_2021 to phase1_2022)
+
```
Comment (jshlee) [inline EventFilter/GEMRawToDigi/test/runGEMUnpacker_cfg.py:166]: This doesn't need to be changed. Can you revert the changes for this file.

**Comment by aloeliger** [comment]
-1

- Unintended commits

**Comment by vshang** [comment]
Closing PR due to unintended commits from merging l1t-integration-v147-CMSSW_13_1_0_pre2 branch.
